lib/matplotlib/artist.py

```diff
@@ -1267,8 +1267,7 @@ def aliased_name(self, s):
 
         if s in self.aliasd:
             return s + ''.join([' or %s' % x
-                                for x
-                                in six.iterkeys(self.aliasd[s])])
+                                for x in sorted(self.aliasd[s])])
```

Comment:
I'm not sure, but there's usually only a few aliases and a whole lot more string manipulation to create the full docstrings so I'd wager not.